File: Project Milestone 2/Final/kafka-python/consumer.py
from kafka import KafkaConsumer

consumer = KafkaConsumer(
    'topic_1',
    bootstrap_servers= ["localhost:9093","localhost:9094","localhost:9095"],
    auto_offset_reset = 'earliest',
    enable_auto_commit = True,
    group_id = 'test',
    )

print("Listening for messages...")

for message in consumer:
    print ("The topic is %s and The message is %s" % (message.topic, message.value))

File: Project Milestone 2/Devorah Jane/kafka/consumer.js
//const Kafka = require("kafkajs").Kafka
const {Kafka} = require("kafkajs")

run();
async function run(){
    try
    {
        const kafka = new Kafka({
            "clientID": "myapp",
            "brokers": [
                "localhost:9093",
                "localhost:9094",
                "localhost:9095"
            ]
        })

        const consumer = kafka.consumer({"groupId": "test"})
        console.log("Connecting.....")
        await consumer.connect()
        console.log("Connected!")
        
        await consumer.subscribe({
            "topic": "Users",
            "fromBeginning": true
        })
        
        await consumer.run({
            "eachMessage": async result => {
                console.log(`RVD Msg ${result.message.value} on partition ${result.partition}`)
            }
        })
 

    }
    catch(ex)
    {
        console.error(`Something bad happened ${ex}`)
    }
    finally{
        
    }


}